Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -165,39 +179,47 @@ public struct SupportedPlatform: Encodable, Equatable {
     /// - Since: First available in PackageDescription 5.0
     ///
     /// - Parameter version: The minimum deployment target that the package supports.
+    /// - Returns: A `SupportedPlatform` instance.
     public static func tvOS(_ version: SupportedPlatform.TVOSVersion) -> SupportedPlatform {
         return SupportedPlatform(platform: .tvOS, version: version.version)
     }
 
     /// Configures the minimum deployment target version for the tvOS platform
     /// using a custom version string.
     ///
-    /// The version string must be a series of two or three dot-separated integers,such as `9.0` or `9.0.1`.
+    /// The version string must be a series of two or three dot-separated
+    /// integers,such as `9.0` or `9.0.1`.
     ///
     /// - Since: First available in PackageDescription 5.0
     ///
-    /// - Parameter versionString: The minimum deployment target as a string representation of two or three dot-separated integers, such as `9.0.1`.
+    /// - Parameter versionString: The minimum deployment target as a string
+    ///     representation of two or three dot-separated integers, such as `9.0.1`.
+    /// - Returns: A `SupportedPlatform` instance.
     public static func tvOS(_ versionString: String) -> SupportedPlatform {
         return SupportedPlatform(platform: .tvOS, version: SupportedPlatform.TVOSVersion(string: versionString).version)
     }
 
-    /// Configures the minimum deployment target version for the watchOS platform.
+    /// Configure the minimum deployment target version for the watchOS
+    /// platform.
     ///
     /// - Since: First available in PackageDescription 5.0
     ///
     /// - Parameter version: The minimum deployment target that the package supports.
+    /// - Returns: A `SupportedPlatform` instance.
     public static func watchOS(_ version: SupportedPlatform.WatchOSVersion) -> SupportedPlatform {
         return SupportedPlatform(platform: .watchOS, version: version.version)
     }
-
-    /// Configures the minimum deployment target version for the watchOS platform
-    /// using a custom version string.
+    
```

Comment:
whitespace-only line

Reply:
Looking at this in context, it seems fine.